From ee1c7996fa8d9f17aca65a1c0ba9ec5c05fb316b Mon Sep 17 00:00:00 2001
From: Demetris Ierokipides <ierokipides.dem@gmail.com>
Date: Sun, 18 Jul 2021 19:38:41 +0300
Subject: [PATCH] MIQi: CPU and GPU regulators syr827 & syr828 output voltage
 can be programmed from 0.7125V to 1.5V through I2C interface. I dont know who
 chose to reduce them to 0.85v-1.35v range and why

---
 arch/arm/boot/dts/rk3288-miqi.dts | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/rk3288-miqi.dts b/arch/arm/boot/dts/rk3288-miqi.dts
index cf54d5fff..0b2facc1b 100644
--- a/arch/arm/boot/dts/rk3288-miqi.dts
+++ b/arch/arm/boot/dts/rk3288-miqi.dts
@@ -137,8 +137,8 @@ vdd_cpu: syr827@40 {
 		fcs,suspend-voltage-selector = <1>;
 		reg = <0x40>;
 		regulator-name = "vdd_cpu";
-		regulator-min-microvolt = <850000>;
-		regulator-max-microvolt = <1350000>;
+		regulator-min-microvolt = <712500>;
+		regulator-max-microvolt = <1500000>;
 		regulator-always-on;
 		regulator-boot-on;
 		regulator-enable-ramp-delay = <300>;
@@ -151,8 +151,8 @@ vdd_gpu: syr828@41 {
 		fcs,suspend-voltage-selector = <1>;
 		reg = <0x41>;
 		regulator-name = "vdd_gpu";
-		regulator-min-microvolt = <850000>;
-		regulator-max-microvolt = <1350000>;
+		regulator-min-microvolt = <712500>;
+		regulator-max-microvolt = <1500000>;
 		regulator-always-on;
 		vin-supply = <&vcc_sys>;
 	};
-- 
2.25.1

